src/apps/common/submodules/chooseModel/chooseModel.js

```diff
@@ -233,7 +233,9 @@ define(function(require) {
 							submodule: 'chooseModel'
 						}));
 				templateDevice.find('.block-footer').slideDown(function() {
-					$('html, body').animate({ scrollTop: templateDevice.find('div.block-device-info div.title-bar').offset().top }, function() {
+					popup.animate({
+						scrollTop: popup.offset().top
```

Comment:
It seems that `popup.offset().top` is the position of the top of the dialog from the viewport's top, so the scrolling is always 49px. However, the text field is focused at the end of the animation, so it gets on screen anyways.

Maybe scroll to the bottom of the dialog instead?

```suggestion
						scrollTop: popup.get(0).scrollHeight
```

Reply:
Also, this is out of the scope of this ticket, but I noticed that I needed to have the apps `desktop-app` and `callthru-us` to be able to load the SmartPBX app. Does that happen to you too?

Reply:
Ah weird, my original selector was `popup.find('.block-device-info')` to scroll to that section specifically, but it doesn't seem like it made it to the PR 😢